parquet-column/src/main/java/org/apache/parquet/CorruptStatistics.java

```diff
@@ -41,6 +41,8 @@
   // the bug involved writing invalid binary statistics, so stats written prior to this
   // fix must be ignored / assumed invalid
   private static final SemanticVersion PARQUET_251_FIXED_VERSION = new SemanticVersion(1, 8, 0);
+  private static final SemanticVersion PARQUET_251_FIXED_START = new SemanticVersion(1, 5, 0, null, "cdh5.5.0", null);
```

Comment:
maybe rename these to `CDH_5_PARQUET_251_FIXED_START` and `CDH_5_PARQUET_251_FIXED_END`